great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -541,31 +544,118 @@ def _display(
         alt.data_transformers.disable_max_rows()
 
-        chart: alt.Chart
-        for chart in themed_charts:
-            chart.display()
+        ipywidgets_dropdown_css: str = f"""
+            <style>
+            .widget-inline-hbox .widget-label {{
+                color: {dropdown_title_color};
+                font-family: {dropdown_font};
+                font-size: {dropdown_font_size}px;
+                font-weight: bold;
+            }}
+            .widget-dropdown > select {{
+                padding-right: 21px;
+                padding-left: 3px;
+                color: {dropdown_text_color};
+                font-family: {dropdown_font};
+                font-size: {dropdown_font_size}px;
+                height: 20px;
+                line-height: {dropdown_font_size}px;
+                background-size: 20px;
+                border-radius: 2px;
+            }}
+            </style>
+        """
+        display(HTML(ipywidgets_dropdown_css))
+
+        dropdown_selection: widgets.Dropdown = widgets.Dropdown(
+            options=chart_titles,
+            description="Select Plot: ",
+            style={"description_width": "initial"},
+            layout={"width": "max-content", "margin": "0px"},
+        )
+
+        # As of 19 July, 2022 there is a DeprecationWarning due to the latest ipywidgets' interaction with
+        # ipykernel versions > 6.0.0. Rather than add a version constraint to ipykernel, we suppress
+        # DeprecationWarnings produced by module ipywidgets.widgets.widget_output
+        warnings.filterwarnings(
+            action="ignore",
+            category=DeprecationWarning,
+            module="ipywidgets.widgets.widget_output",
+        )
+        widgets.interact(
+            DataAssistantResult._display_chart_from_dict,
+            display_chart_dict=widgets.fixed(display_chart_dict),
+            chart_title=dropdown_selection,
+        )
+
+    @staticmethod
+    def _display_chart_from_dict(
+        display_chart_dict: Dict[str, Union[alt.Chart, alt.LayerChart]],
+        chart_title: str,
+    ) -> None:
+        display_chart_dict[chart_title].display()
+
+    @staticmethod
+    def _get_chart_titles(charts: List[alt.Chart]) -> List[str]:
+        chart_titles: List[str] = []
+        chart_title: Optional[str]
+        for chart in charts:
+            chart_title = None
+            try:
+                chart_title = chart.title.text
+            except AttributeError:
+                for layer in chart.layer:
+                    try:
+                        chart_title = layer.title.text
+                        break
+                    except AttributeError:
+                        continue
+
+            if chart_title is not None:
+                chart_titles.append(chart_title)
+            else:
+                raise ge_exceptions.DataAssistantResultExecutionError(
+                    "All DataAssistantResult charts must have a title."
+                )
```

Comment:
```suggestion
            if not chart_titles:
                raise ge_exceptions.DataAssistantResultExecutionError(
                    "All DataAssistantResult charts must have a title."
                )
           chart_titles.append(chart_title)
```

Using a guard clause here might make this easier to read.
https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88

Reply:
I came up with something even simpler:

```
            if chart_title is None:
                raise ge_exceptions.DataAssistantResultExecutionError(
                    "All DataAssistantResult charts must have a title."
                )

            chart_titles.append(chart_title)
```